humble-programming.org

```diff
@@ -0,0 +1,29 @@
+* Abstract
+One who knows everything can learn nothing. The most important lesson I've
+learned through my career has been to consistently check my own ego and learn
+from everyone around me. Being humble helps us better connect to others, make
+stronger technical decisions, and most significantly, have a better relationship
+with ourselves.
+
+It surprised me to need to learn that, because I had always thought of myself as
+humble. However, egotism is deceptive and it crops up when we least expect it.
+I'll share several situations in career and how being humble improved the social
+and technical outcomes. My hope is to share the mistakes I've made and extract
+the lessons they taught me. Attendees will come away with specific strategies
+for recognizing when they their actions are being influenced by ego and how to
+temper that effect.
+* Ideas
+** Other people's time is more important than yours
+- They have more pressing things to do
+- PR reviews
+- Ticket finishing and testing
+** Some people are bristly
+- Don't get defensive
+- Marcus Aurelius quote about people being assholes
```

Comment:
I'm curious what this quote is. :)

Reply:
I think it's something like this 😄
> When you wake up in the morning, tell yourself: the people I deal with today will be meddling, ungrateful, arrogant, dishonest, jealous and surly. They are like this because they can't tell good from evil. But I have seen the beauty of good, and the ugliness of evil, and have recognized that the wrongdoer has a nature related to my own - not of the same blood and birth, but the same mind, and possessing a share of the divine. And so none of them can hurt me. No one can implicate me in ugliness. Nor can I feel angry at my relative, or hate him. We were born to work together like feet, hands and eyes, like the two rows of teeth, upper and lower. To obstruct each other is unnatural. To feel anger at someone, to turn your back on him: these are unnatural

https://www.goodreads.com/work/quotes/31010